one-ui/src/main/java/com/marklogic/hub/oneui/managers/ModelManager.java

```diff
@@ -182,18 +182,6 @@ public JsonNode getLatestJobInfo(String modelName) {
     }
 
     JsonNode getJobInfoFromDB(DatabaseClient dbClient, String modelName) {
-        JsonNode node = null;
-        try {
-            node = JobInfo.on(dbClient).getLatestJobData(modelName);
-        }
-        catch (MarkLogicServerException e) {
-            /*
-             * As MarkLogic server throws a FailedRequestException for both data-service module not found
-             * and if the user is unauthorized to access the module; we need to fail silently and log the
-             * issue.
-             */
-            logger.error(e.getMessage());
-        }
-        return node;
+        return JobInfo.on(dbClient).getLatestJobData(modelName);
     }
```

Comment:
Do we have test for this in place?

Reply:
Yes, the data service is tested via ml-unit-test

Reply:
This PR just removes the try/catch block surrounding the code.